netci.groovy

```diff
@@ -1176,14 +1180,27 @@ combinedScenarios.each { scenario ->
                                         else if (architecture == 'x64') {
                                             if (isLongGc(scenario)) {
                                                 buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture} longgctests sequential"
-                                            } else {
+                                            } 
+                                            else {
                                                 buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture}"
                                             }
+                                        }                                        
+                                        else if (architecture == 'x86ryujit') {
+                                            if (isLongGc(scenario)) {
+                                                buildCommands += "tests\\runtest.cmd ${lowerConfiguration} ${architecture} longgctests sequential"
```

Comment:
Wouldn't we also need the testenv here? Otherwise, we would just be running with legacy backend in what we are calling a RyuJIT scenario